modin/api/partition_api.py

```diff
@@ -78,3 +81,77 @@ def _unwrap_partitions(oid):
             part.coalesce(bind_ip=bind_ip).unwrap(squeeze=True, bind_ip=bind_ip)
             for part in partitions
         ]
+
+
+def create_df_from_partitions(partitions, axis):
```

Comment:
My concern with the name is that we will have more than just `pandas` API in the near future. How will this API help in those situations?

Reply:
My first thoughts on this is to add something like that: `convert_df_to_smth`. But I would leave as it is now and would think on this later when we will be near to that API.

Reply:
From discussion with @devin-petersohn , we chose the following locations for the functions:
```python
from modin.developer_api.pandas import from_partitions, uwnrap_partitions # 1
from modin.developer_api.pandas.dataframe import from_partitions, unwrap_partitions # 2
from modin.developer.pandas import from_partitions, uwnrap_partitions # 3
from modin.developer.pandas.dataframe import from_partitions, unwrap_partitions # 4
```
@vnlitvinov , @dchigarev , what do you think? What is more appropriate?

Reply:
I don't get the presence of `developer` in any of these. I understand that we intend this API to be used by Modin (or Modin-using products) devs, but I still don't like that in the name. What was wrong with simple `modin.api`?..

Adding `.pandas` sublevel (like in option # 3) seems appropriate and aligned with `modin.pandas` naming, though, so I like that. Hence I would suggest `from modin.api.pandas import fooo` option :) Plus to place real functions in `.pandas.dataframe` submodule (to show where they logically belong) but import them in `.pandas` parent one to ease access (so a combination of options 4 and 3).

Reply:
I am okay with this. Fixed.